src/common/event/EventSystem.hpp

```diff
@@ -79,14 +61,31 @@ class EventSystem
  /**
   * @brief Get the instance of the event system
   * @details This function returns an instance of the event system
-  * @return EventSystem*, the instance of the event system
+  * @return EventSystem&, the instance of the event system
   * @version 0.1.0
   * @since 0.1.0
-  * @author Marius PAIN
+  * @author Aubane NOURRY
   */
- static EventSystem* getInstance();
+ static EventSystem &getInstance();
 
 private:
+ /**
+  * @brief Default constructor for EventSystem private to prevent creation.
+  * @details Initializes the EventSystem with no listeners.
+  * @version v0.1.0
+  * @since v0.1.0
+  * @author Aubane NOURRY
```

Comment:
You don't seemed to have touch at the implementation of the constructor just at its privacy. So for the next time, you can just change `@author` into `@authors` x)

![giphy (4)](https://github.com/user-attachments/assets/68912641-3ba9-44c4-a5df-e9e7c3f0654d)

Reply:
I didn't know it existed! I changed to @authors